### PR TITLE
EZP-21650 AuthorsFieldType BDD scenario

### PR DESCRIPTION
**Pull request by miguelcleverti**
# EZP-2477 BDD scenario
BDD scenarios for issue EZP-21650
jira: https://jira.ez.no/browse/EZP-21650

requires: #181

### CODE REVIEW

File: Features/Stories/Fieldtypes/EZP-2477-AuthorsFieldType.feature [hunk 1]
```diff
@@ -0,0 +1,97 @@
+Feature: Test the validations done on Author fieldtype from Editorial Interface
+
+    @javascript
+    Scenario: Message when using empty author field
+        Given I go to PlatformUI app with username "admin" and password "publish"
```
Comment (pcardiga) [inline Features/Stories/Fieldtypes/EZP-2477-AuthorsFieldType.feature]: All the administrator logins could be replaced by sentences such:
        Given I am logged in as an administrator

And the implementation would login as admin/publish.
Option: We could have a before-scenario hook where the login is made. Either that or including this line would fit the purpose o the login.

File: Features/Stories/Fieldtypes/EZP-2477-AuthorsFieldType.feature [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        And a Content Type exists with identifier "Author" in Group with identifier "Content" with fields:
+            | identifier | Type     | Name   |
+            | name       | ezstring | Name   |
+            | author     | ezauthor | Author |
+        And I click on the menu zone "Platform"
+        And I click on the "Content structure" link
+        And I click on the side menu option "Create a content"
+        And I click on the content type "Author"
+        And I fill in "Name" with "AuthorTest"
```
Comment (pcardiga) [inline Features/Stories/Fieldtypes/EZP-2477-AuthorsFieldType.feature]: We could shorten the sentences:
"
        And I click on the menu zone "Platform"
        And I click on the "Content structure" link
        And I click on the side menu option "Create a content"
        And I click on the content type "Author"
        And I fill in "Name" with "AuthorTest"
"

With 
        And I create a content of content type "Author" with
            | name       | author | email |
            | AuthorTest |        |       | 

Where the creation of the content would follow all the steps described before.

Comment (andrerom) [inline Features/Stories/Fieldtypes/EZP-2477-AuthorsFieldType.feature]: +1

File: Features/Stories/Fieldtypes/EZP-2477-AuthorsFieldType.feature (hunk 1 above)
Comment (andrerom) [inline Features/Stories/Fieldtypes/EZP-2477-AuthorsFieldType.feature]: +1 (more or less, but we did try something like this last year, lets make sure it does not grow into the same abstract level)

**Comment by andrerom** [comment]
Besides those commenets:
@pcardiga We need to define what we actually want to cover in BDD, cause all of these PR's focus on the issue, while they should focus on covering the features (see files names), this will lead to way to many BDD tests and also would be bad way to organize them,

**Pull request closed without merging**